### PR TITLE
Xerces-C: some supporting libs added.

### PR DESCRIPTION
**Pull request by miqlas**
Solves the missing symbol: "connect"

### CODE REVIEW

File: dev-libs/xerces_c/xerces_c-3.1.4.recipe [hunk 1]
```diff
@@ -64,7 +74,10 @@ BUILD_PREREQUIRES="
 BUILD()
 {
 	autoreconf -fi
-	runConfigure ./configure
+	LDFLAGS="-lnetwork -lbsd" \
```
Comment (korli) [inline dev-libs/xerces_c/xerces_c-3.1.4.recipe]: You should patch m4/xerces_netaccessor_selection.m4 line 29 instead.

**Comment by korli** [comment]
Please precise the need to enable curl, when a native "Sockets-based NetAccessor" option seems to be available (I'd think more features available) in your commit message.
Also the choice of iconv vs ICU.

**Comment by miqlas** [comment]
curl removed.

File: dev-libs/xerces_c/xerces_c-3.1.4.recipe (hunk 1 above)
Comment (miqlas) [inline dev-libs/xerces_c/xerces_c-3.1.4.recipe]: Already did it.

**Pull request merged**